### PR TITLE
GO-LIVE VALUES

### PR DESCRIPTION
**Pull request by andygray**
@votive @jamesmorgan - I think I might be an hour out with DST etc. Thoughts?

See `PixieCrowdsale.dates.test.js`

### CODE REVIEW

File: contracts/pixie/PixieCrowdsale.sol [hunk 1]
```diff
@@ -17,34 +17,31 @@ contract PixieCrowdsale is Crowdsale, Pausable {
...
+  uint256 public privateSaleCloseTime = 1533113999;
 
-  // Monday, 1st October 2018 00:00:00
-  uint256 public preSaleCloseTime = 1538348400;
+  // Monday, October 1, 2018 9:59:59 AM GMT+01:00
+  uint256 public preSaleCloseTime = 1538384399;
 
-  // Friday, 21st December 2018 00:00:00
-  uint256 public closingTime = 1545350400;
+  // Wednesday, October 31, 2018 9:59:59 AM GMT+00:00
```
Comment (jamesmorgan) [inline contracts/pixie/PixieCrowdsale.sol]: `GMT+01:00` ?

File: test/truffle/pixie/PixieCrowdsale.dates.test.js [hunk 2]
```diff
@@ -26,47 +26,73 @@ contract('PixieCrowdsale (real dates)', function ([owner, investor, wallet]) {
     this.crowdsale = await PixieCrowdsale.new(wallet, this.token.address);
   });
 
-  describe('get correct ICO dates', async function () {
+  describe.only('get correct ICO dates', async function () {
```
Comment (jamesmorgan) [inline test/truffle/pixie/PixieCrowdsale.dates.test.js]: `.only`

File: contracts/pixie/PixieCrowdsale.sol (hunk 1 above)
Comment (andygray) [reply contracts/pixie/PixieCrowdsale.sol]: might be wrong - will double check next week

Comment (andygray) [reply contracts/pixie/PixieCrowdsale.sol]: I think that is right

**Pull request merged**